### DISCUSSION TITLE
search results occurances

### DISCUSSION THREAD

**Discussion by igalM** [stephanrauh/ngx-extended-pdf-viewer#1436] [Q&A]
Hi,
Lets say i am searching multiple terms, in this case - 'First' & 'Preliminary'.
I have an array of elements like this:
`
[
{ pageNumber: 5, occurance: 1, text: 'First'},
{ pageNumber: 5, occurance: 1, text: 'Preliminary'},
{ pageNumber: 5, occurance: 2, text: 'Preliminary'}
]
`
I want to highlight only the 1st result of the word 'First', and the 1st + 2nd result of the word 'Preliminary'.
Is there any way to achieve this result?
And can it be achieved across all the document pages?

<img width="708" alt="Screen Shot 2022-06-30 at 12 11 59" src="https://user-images.githubusercontent.com/40319153/176639779-3e608258-6dc8-4633-b90c-deb49006e934.png">

**Comment by stephanrauh** [comment]
Currently, that's not possible. Maybe I can add a user hook allowing you to modify the list of search results. Let me think about it a while. Maybe this is useful for other use-cases, too.
